style.css

```diff
@@ -688,3 +739,587 @@
+  .close-dash {
+    display: none;
+  }
+
+  .various-Project-heading-4,
+  .various-Project-heading-2-desktop,
+  .various-Project-heading-5-desktop {
+    display: flex;
+    margin-top: 178px;
+    margin-left: 16px;
+    margin-right: 16px;
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 2rem;
+    line-height: 2.25rem;
+    color: #fff;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .various-Project-heading-3-desktop,
+  .various-Project-heading-6-desktop {
+    display: flex;
+    margin-top: 221px;
+    margin-left: 16px;
+    margin-right: 16px;
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 2rem;
+    line-height: 2.25rem;
+    color: #fff;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .about-section {
+    background: #262626;
+    background-image: url(./images/background/about-section\ background.png);
+    background-repeat: no-repeat;
+    background-position: top right;
+    background-position-y: 8.32vh;
+    background-size: 40.66vw;
+    height: 1195px;
+  }
+
+  .about-heading {
+    width: 566px;
+    height: 78px;
+    margin-top: 130px;
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 4.5rem;
+    line-height: 5.125rem;
+    letter-spacing: 0.37px;
+    color: #f55800;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    margin-left: 142px;
+    margin-right: 142px;
+  }
+
+  .about-text {
+    width: 566px;
+    height: 148px;
+    margin-top: 24px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 1.25rem;
+    line-height: 2rem;
+    color: #f4f5f7;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+    margin-left: 142px;
+    margin-right: 142px;
+  }
+
+  .resume {
+    margin-top: 24px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    gap: 10px;
+    width: 161px;
+    height: 48px;
+    background: #f55800;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 1rem;
+    line-height: 1.5rem;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #fff;
+    margin-left: 142px;
+    margin-right: 142px;
+    flex: none;
+    order: 2;
+    flex-grow: 0;
+    border: none;
+  }
+
+  .horizontal-line {
+    margin-top: 139px;
+    margin-left: 142px;
+    margin-right: 142px;
+    margin-bottom: 146.73px;
+  }
+
+  .skill-list {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-around;
+  }
+
+  .head-skill-list {
+    text-align: center;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 2rem;
+    line-height: 2.25rem;
+    margin-bottom: 32px;
+    color: #fff;
+  }
+
+  .semi-list {
+    list-style: none;
+    text-decoration: none;
+    text-align: center;
```

Comment:
- [ ] I think it's better to align this list to the left following the Figma template. It's a project requirement.

Reply:
done